launch/morse.launch

```diff
@@ -1,11 +1,25 @@
 <launch>
     <param name="/use_sim_time" value="true" />
-    <rosparam command="load" file="$(find iarc7_simulator)/param/morse.yaml" />
+    <arg name="platform" default="sim" doc="string representing the platform" />
 
-    <include file="$(find iarc7_simulator)/launch/static_transforms_sim.launch" />
+    <group if="$(eval str(arg('platform')) == 'sim')" >
+        <rosparam command="load" file="$(find iarc7_simulator)/param/morse.yaml" />
+        <include file="$(find iarc7_simulator)/launch/static_transforms_sim.launch" />
 
-    <node name="$(anon morse_launcher)" pkg="iarc7_simulator"
+        <node name="$(anon morse_launcher)" pkg="iarc7_simulator"
         type="morse_launcher.sh" cwd="node" />
 
-    <node name="simulator_adapter" pkg="iarc7_simulator" type="simulator_adapter.py" />
+        <node name="simulator_adapter" pkg="iarc7_simulator" type="simulator_adapter.py" />
+    </group>
+
+    <group if="$(eval str(arg('platform')) == 'sim_1.5')" >
+        <rosparam command="load" file="$(find iarc7_simulator)/param/morse-prototype.yaml" />
+        <include file="$(find iarc7_simulator)/launch/static_transforms_prototype_sim.launch" />
+
+        <node name="$(anon morse_launcher)" pkg="iarc7_simulator"
```

Comment:
The two nodes here are exactly the same, can we put them outside these group statements?